Doc/whatsnew/3.13.rst

```diff
@@ -1008,109 +1143,250 @@ Optimizations
+  * :mod:`!nis`.
+    (Contributed by Victor Stinner in :gh:`104773`.)
 
-CPython will run with the :term:`global interpreter lock` (GIL) disabled when
-configured using the ``--disable-gil`` option at build time. This is an
-experimental feature and therefore isn't used by default. Users need to
-either compile their own interpreter, or install one of the experimental
-builds that are marked as *free-threaded*.
+  * :mod:`!nntplib`:
+    the :pypi:`nntplib` PyPI project can be used instead.
+    (Contributed by Victor Stinner in :gh:`104773`.)
 
-Free-threaded execution allows for full utilization of the available
-processing power by running threads in parallel on available CPU cores.
-While not all software will benefit from this automatically, programs
-designed with threading in mind will run faster on multicore hardware.
+  * :mod:`!ossaudiodev`: use the
+    `pygame project <https://www.pygame.org/>`_ for audio playback.
+    (Contributed by Victor Stinner in :gh:`104780`.)
 
-Work is still ongoing: expect some bugs and a substantial single-threaded
-performance hit.
+  * :mod:`!pipes`: use the :mod:`subprocess` module instead.
+    (Contributed by Victor Stinner in :gh:`104773`.)
 
-The free-threaded build still supports optionally running with GIL enabled at
-runtime using the environment variable :envvar:`PYTHON_GIL` or the command line
-option :option:`-X gil`.
+  * :mod:`!sndhdr`: use the projects :pypi:`filetype`,
+    :pypi:`puremagic`, or :pypi:`python-magic` instead.
+    (Contributed by Victor Stinner in :gh:`104773`.)
 
-* Use :func:`!sys._is_gil_enabled` to determine if the :term:`GIL` is enabled.
+  * :mod:`!spwd`:
+    the :pypi:`python-pam` project can be used instead.
+    (Contributed by Victor Stinner in :gh:`104773`.)
 
-* Use ``sysconfig.get_config_var("Py_GIL_DISABLED")`` to identify CPython
-  builds configured with ``--disable-gil``.
+  * :mod:`!sunau`.
+    (Contributed by Victor Stinner in :gh:`104773`.)
 
-C-API extensions need to be built specifically for the free-threaded build.
+  * :mod:`!telnetlib`, use the projects :pypi:`telnetlib3 ` or
+    :pypi:`Exscript` instead.
+    (Contributed by Victor Stinner in :gh:`104773`.)
 
-* Extensions that support running with the :term:`GIL` disabled should use
-  the :c:data:`Py_mod_gil` slot. Extensions using single-phase init should use
-  :c:func:`PyUnstable_Module_SetGIL` to indicate whether they support running
-  with the GIL disabled. Importing C extensions that don't use these mechanisms
-  will cause the GIL to be enabled unless the GIL was explicitly disabled with
-  the :envvar:`PYTHON_GIL` environment variable or the :option:`-X gil=0`
-  option.
+  * :mod:`!uu`: the :mod:`base64` module is a modern alternative.
+    (Contributed by Victor Stinner in :gh:`104773`.)
 
-* pip 24.1b1 or newer is required to install packages with C extensions in the
-  free-threaded build.
+  * :mod:`!xdrlib`.
+    (Contributed by Victor Stinner in :gh:`104773`.)
+
+2to3
```

Comment:
```suggestion
2to3 and tkinter.tix
```
Move tkinter entry here

Or split 'Removed Modules and APIs' into 2 headers.  There are no module removals scheduled for next few releases, so 'Removed Modules' will not be needed for awhile.

Reply:
I've merged the PEP 594 section with the other removals.